### PR TITLE
add files for full website for PR

### PR DESCRIPTION
**Pull request by supreeti**
1. Create a math magician website🌐 consisting of 3 pages: Home, Calculator, and Quote.✅
2. The general layout should match the given wireframes 🏠.✅
3. Add your own styles to improve the look and feel 🧑‍🎨 . ✅
4. Use React basic styling (inline styles, importing CSS and pre-processed CSS files). ✅
5. For every page, create a . ✅
6. Make navigation links for all 3 routes are displayed on each page. ✅
7. Use [React Router](https://reactrouter.com/) to make the routes work. ✅

8. **Please Note that i am not able to resolve linter error for missing file extention for 1 file.**

### CODE REVIEW

**Review by Shedrack-Sunday — CHANGES_REQUESTED** [review]
Hi @supreeti 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request 🔴 

You have submitted a project, that has some failing linters. Kindly fix them before sending for reviews

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: package.json [hunk 1]
```diff
@@ -10,6 +10,7 @@
     "prop-types": "^15.8.1",
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
+    "react-router-dom": "^6.14.1",
     "react-scripts": "5.0.1",
     "web-vitals": "^2.1.4"
```
Comment (elisha2kyakpo1) [inline package.json:15]: - [ ] The app works as expected, but there is an error that shows up on the screen. Kindly fix this issue and provide enough information required for running your project in the readme file.

error

**Review by elisha2kyakpo1 — CHANGES_REQUESTED** [review]
Hi 👋 @supreeti, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

✔️ The PR title and description are good and professional.
✔️ No linter errors.
✔️ Descriptive readme file.
✔️ Calculator works👏
👍 Quotes are fetched and displayed nicely. 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by DeliceLydia — APPROVED** [review]
## Hi @supreeti, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- Addressed changes requested



### Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**